src/common/NavbarItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function NavbarItem(props) {
+    const history = useHistory();
+    const url = props.title.toLowerCase();
+    return <div className="navbar-list-item" onClick={e => history.push(`/${url}`)}>
```

Comment:
I prefer that the onClick will use a function and not use direct implementation here and in other places.
One of the reasons is if the functionality will become longer, you will want to wrap with try/catch and more reasons if we will think about it